### DISCUSSION TITLE
Bruteforce on GPU

### DISCUSSION THREAD

**Discussion by FLozyXD** [AloneLiberty/FlipperNested#9] [Q&A] [answered]
Is there any way of brute forcing keys with a graphics card? I think it has more power than processor. Or it is not so simple?

**Comment by AloneLiberty** [comment, marked as answer]
Not simple and there is no point for that, regular CPU can recover in 10-15 minutes (even faster with SIMD, but we are missing it, with them recovery should we even faster)
